pkg/api/testapi/testapi.go

```diff
@@ -101,6 +165,6 @@ func ResourcePathWithPrefix(prefix, resource, namespace, name string) string {
 // Returns the appropriate path for the given resource, namespace and name.
 // For example, this is of the form:
 // /api/v1/namespaces/foo/pods/pod0 for v1.
-func ResourcePath(resource, namespace, name string) string {
-	return ResourcePathWithPrefix("", resource, namespace, name)
+func ResourcePath(resource, namespace, name string, group ...string) string {
```

Comment:
Same as above two comments. I really think group should be first and non-optional.